CLI: Error creating CLI release from master branch

### Parent Issue

### Problem Statement
When attempting to create a CLI release for the from the master branch, the following error occurs on the `Tag release version` step:

```
error: pathspec 'master' did not match any file(s) known to git
Error: Process completed with exit code 1.
```

### Steps to Reproduce
1. Navigate to the `Github Actions` of `core` repo.
2. Run the Github Action `dotCLI Release` from the `master` branch.

### Acceptance Criteria
- The CLI release creation process from the `master` branch should execute without errors.
- Tags should be successfully created for releases from the `master` branch.

### dotCMS Version
N/A

### Proposed Objective
Quality Assurance

### Proposed Priority
Priority 2 - Important

### External Links
N/A

### Assumptions & Initiation Needs
N/A

### Quality Assurance Notes & Workarounds
N/A

### Sub-Tasks & Estimates
N/A

--- a/.github/workflows/cli-release-process.yml
+++ b/.github/workflows/cli-release-process.yml
@@ -315,18 +315,6 @@ jobs:
         run: |
           tools/dotcms-cli/mvnw -B -Prelease jreleaser:full-release -DartifactsDir=artifacts -Djreleaser.git.root.search=true -pl :dotcms-cli-parent -Dmaven.plugin.validation=VERBOSE
 
-      - name: 'Setup git config'
-        run: |
-          git config user.name "${{ secrets.CI_MACHINE_USER }}"
-          git config user.email "dotCMS-Machine-User@dotcms.com"
-
-      - name: 'Tag release version in ${{ github.ref }}'
-        if: ${{ !contains(github.event.inputs.version, 'SNAPSHOT') }}
-        run: |
-          git checkout ${{ needs.precheck.outputs.HEAD }}
-          git tag -a "dotcms-cli${{ needs.precheck.outputs.RELEASE_VERSION }}" -m "Release dotCLI version ${{ needs.precheck.outputs.RELEASE_VERSION }}"
-          git push origin "dotcms-cli${{ needs.precheck.outputs.RELEASE_VERSION }}"
-
   publish-npm-package:
     name: "Publish NPM Package"
     if: success()  # Run only if explicitly indicated and successful

--- a/tools/dotcms-cli/jreleaser.yml
+++ b/tools/dotcms-cli/jreleaser.yml
@@ -20,12 +20,12 @@ project:
 
   snapshot:
     pattern: .*-SNAPSHOT
-    label: 'dotcms-cli{{projectVersion}}'
+    label: 'dotcms-cli-{{projectVersion}}'
     fullChangelog: true
 
 release:
   github:
-    tagName: 'dotcms-cli{{projectVersion}}'
+    tagName: 'dotcms-cli-{{projectVersion}}'
     releaseName: 'dotcms-cli - {{projectVersion}}'
     overwrite: false
     changelog:

